pkg/accountdb/user/repository_test.go

```diff
@@ -0,0 +1,234 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package user
+
+import (
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"storj.io/storj/pkg/accountdb"
+	"testing"
+	"time"
+)
+
+// test values
+var (
+	lastName = "lastName"
+	email = "email@ukr.net"
+	pass = "pass"
+	passValid = "123456"
+	name = "name"
+	newName = "newName"
+	newLastName = "newLastName"
+	newEmail = "newEmail@ukr.net"
+	newPass = "newPass"
+)
+
+func TestRepository(t *testing.T) {
+
+	accountdb.NewDbFactory("file::memory:?mode=memory&cache=shared", "sqlite3")
+
+	db, err := accountdb.GetDb()
+
+	if err != nil {
+
+	}
+
+	defer func() {
+		conn, _ := db.GetConnection()
+		conn.Close()
+	}()
+
+	repository := NewRepository()
+
+	cases := []struct {
```

Comment:
This cases array doesn't make sense. Use sub-tests instead, if you want separation. Or just write them in blocks.